go.mod

```diff
@@ -12,8 +12,8 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/rhysd/actionlint v1.6.23
 	go.opencensus.io v0.24.0
-	go.viam.com/api v0.1.98
-	go.viam.com/rdk v0.2.32
+	go.viam.com/api v0.1.101
+	go.viam.com/rdk v0.2.33-0.20230331161745-2890e80bfe95
```

Comment:
[note] needs to be updated as RDK 0.2.32 still has old endpoints and with the API update, these would no long be available to that version of rdk.